inference-engine/src/inference_engine/shape_infer/ie_detectionoutput_onnx_shape_infer.hpp

```diff
@@ -1,46 +0,0 @@
-// Copyright (C) 2018-2020 Intel Corporation
-// SPDX-License-Identifier: Apache-2.0
```

Comment:
have we updated IR reader to read experimental `ExperimentalDetectronDetectionOutput` from older IR v10 versions to this new ngraph `ExperimentalDetectronDetectionOutput` operation? Otherwise, BW compatibility is broken if we remove this shape inference.

Reply:
Yes, we have: see lines 505-509 in `inference-engine/src/readers/ir_reader/ie_ir_parser.cpp`.